src/components/ProjectsList/__tests__/ProjectsList.test.tsx

```diff
@@ -1,14 +1,15 @@
 import { ProjectsList } from '../ProjectsList';
 import { MemoryRouter } from 'react-router-dom';
 import { render, screen } from '@testing-library/react';
-import renderer from 'react-test-renderer';
+
+jest.mock('../../../services/projectService');
 
 describe('display ProjectList component', () => {
   let mockProjectsRequest: any;
   let mockProjects: any;
 
   async function loadMocks() {
-    mockProjectsRequest = await import('./data/mock-projects-request.json');
+    mockProjectsRequest = await import('../../../services/__mocks__/projects-mock.json');
```

Comment:
Using `const mockProjectsRequest` and removing `let mockProjectsRequest: any;` should be enough.

Do we want to keep the same convention as on the other places? Then maybe `projectsRequestMock` and `projectsMock` would be better.